Description: 0003-remove-custom-mprotect-function.patch
 This patch removes an fs-uae-internal implementation of the glibc function
 mprotect(). This implementation causes fa-uae to segfault when using
 recent versions of the proprietary nVidia driver. nVidia has investigated
 into the issue and identified the problem. Simply removing mprotect()
 from the fs-uae code base results in the code using glibc's implementation
 of mprotect() and fixes the segfault.

--- fs-uae-2.4.1+ds.orig/src/od-fs/mman.cpp
+++ fs-uae-2.4.1+ds/src/od-fs/mman.cpp
@@ -881,15 +881,6 @@ STATIC_INLINE uae_key_t find_shmkey (uae
     return result;
 }
 
-int mprotect (void *addr, size_t len, int prot)
-#ifdef PANDORA    
-    __THROW
-#endif
-{
-    int result = 0;
-    return result;
-}
-
 void *uae_shmat (int shmid, void *shmaddr, int shmflg)
 {
     write_log("uae_shmat shmid %d shmaddr %p, shmflg %d natmem_offset = %p\n",
